corehq/apps/hqwebapp/templates/hqwebapp/partials/ko_select_toggle.html

```diff
@@ -6,7 +6,12 @@
                                attr: {selected: $data.selected}"></option>
     </select>
     <div class="btn-group-separated" data-bind="foreach: options">
-      <button class="btn btn-default"
-              data-bind="text: $data.text, css: {active: $data.selected}, click: click"></button>
+      <button class="btn btn-default btn-disabled"
+              data-bind="text: $data.text,
+                         css: {
+                           active: $data.selected,
+                           disabled: $parent.disabled,
```

Comment:
This is a little ideological, but if the root is disabled I think the select element above should also be set to disabled.